ale_linters/markdown/mdl.vim

```diff
@@ -0,0 +1,41 @@
+" Author: Steve Dignam <steve@dignam.xyz>
+" Description: Support for mdl, a markdown linter
+
+if exists('g:loaded_ale_linters_markdown_mdl')
+    finish
+endif
+
+let g:loaded_ale_linters_markdown_mdl = 1
+
+function! ale_linters#markdown#mdl#Handle(buffer, lines) abort
+    " matches: '(stdin):173: MD004 Unordered list style'
+    let pattern = ':\(\d*\): \(.*\)$'
```

Comment:
I found the problem with the build. Piping the errors through `sed` to fix the paths was messing up the error codes. I switched the Makefile to using Bash, and used the `pipefail` option to fix that. If the pull request was rebased again, it should fail the build now.